abstract-par/Control/Monad/Par/Class.hs

```diff
@@ -144,26 +144,28 @@ class ParFuture ivar m  => ParIVar ivar m | m -> ivar where
 --   in the /local/ @Par@ thread.  That is, at @fork@ points it is
 --   necessary to give the child computation a separate set of stream
 --   cursors so that it observes the same sequences as the parent.
+{-
```

Comment:
Is this not used anywhere? We probably shouldn't have commented-out code.

Reply:
The only reference is to `ParChan not released yet` from 2012. (https://github.com/simonmar/monad-par/blob/master/monad-par-extras/Control/Monad/Par/State.hs#L71). Should I remove it, or add a `#if 0` like in the other module?

Reply:
It's also not exported by the module https://github.com/simonmar/monad-par/blob/master/abstract-par/Control/Monad/Par/Class.hs#L33 (also from 2012)